### PR TITLE
Issue #13345: Enable AvoidNestedBlockCheckExamplesTest.java

### PR DESCRIPTION
**Pull request by piyush-1234**
Issue: #13345

### CODE REVIEW

**Comment by piyush-1234** [comment]
done @romani

File: src/xdocs/checks/blocks/avoidnestedblocks.xml [hunk 1]
```diff
@@ -112,23 +113,23 @@ public void foo() {
...
+    switch (a) {
+      case 1: {                    // OK
         System.out.println(&quot;Case 1&quot;);
         break;
       }
-    case 2:
-      System.out.println(&quot;Case 2&quot;);     // OK
-      break;
+      case 2:
+        System.out.println(&quot;Case 2&quot;);     // OK
```
Comment (romani) [inline src/xdocs/checks/blocks/avoidnestedblocks.xml:130]: Please remove OK comments, we allowing them only if it has extra text to explain why it is ok.
You have few in your examples.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/checks/blocks/avoidnestedblocks.xml (hunk 1 above)
Comment (piyush-1234) [reply src/xdocs/checks/blocks/avoidnestedblocks.xml]: > Please remove OK comments, we allowing them only if it has extra text to explain why it is ok. You have few in your examples.
Done.

**Review by romani — APPROVED** [review]
Ok to merge

**Review by rnveach — APPROVED** [review]

**Pull request merged**